src/ts/components/playbacktimelabel.ts

```diff
@@ -132,7 +134,15 @@ export class PlaybackTimeLabel extends Label<PlaybackTimeLabelConfig> {
       }
     };
 
+    let playbackReadyHandler = () => {
+      // Set time format depending on source duration
+      this.timeFormat = Math.abs(player.isLive() ? player.getMaxTimeShift() : player.getDuration()) >= 3600 ?
```

Comment:
It seems to me that a format string could be used to change between various string formats. In the latest release, the only formats are _mm:ss_ and _hh:mm:ss_, but the first doesn't map to the variable's range (there could be time quantities > 1 hour). It's not clear to me then in what use case this would apply...

Is the use case that the user of the library could select _mm:ss_ even if time is > 1 hour where the Player logically should trigger to display the correct value (_hh:mm:ss_)?

I would ask what exactly this time format should do - 
- to remove the hours field when = 0?
- to allow the user to select a time format? (hh:mm:ss, total minutes:seconds, total seconds, etc)
- to ignore the hour value?

Reply:
one note on the last comment

> Is the use case that the user of the library could select mm:ss even if time is > 1 hour where the Player logically should trigger to display the correct value (hh:mm:ss)?

If the use case is that the user of the library controls this format, then would it be inconsistent if the Bitmovin UI would change this format when it's value is out of (the format's) range? The following program trajectory could occur:

- The user uses `PlaybackTimeLabel.setTimeFormat()` to set _mm:ss_
- The duration changes and is > 1h
- The component observes this and corrects the format to _hh:mm:ss_

So the user would not be able to control the format if the component would correct the duration during the program trajectory.